src/lib/Client/Deepl.php

```diff
@@ -82,13 +91,32 @@ private function normalized(string $languageCode): string
             return $code;
         }
 
+        if ('zh_TW' === $languageCode) {
+            return 'ZH-HANT';
+        }
+
+        if ('zh_CN' === $languageCode || 'zh_HK' === $languageCode || $code === 'ZH') {
+            return 'ZH-HANS';
+        }
+
+        if ('en_GB' === $languageCode) {
+            return 'EN-GB';
+        }
+
+        if ('EN' === $code) {
+            return 'EN-US';
+        }
+
+        if ('pt_BR' === $languageCode) {
+            return 'PT-BR';
+        }
+
+        if ('PT' === $code) {
+            return 'PT-PT';
+        }
```

Comment:
Can we introduce here some conversion map? It should be even parameter injected into service ?